packages/babel-helpers/src/helpers/get.ts

```diff
@@ -0,0 +1,38 @@
+/* @minVersion 7.0.0-beta.0 */
+
+import _superPropBase from "./superPropBase.ts";
+
+// https://tc39.es/ecma262/multipage/reflection.html#sec-reflect.get
+//
+//  28.1ak.5 Reflect.get ( target, propertyKey [ , receiver ] )
+
+type Object = {
+  __proto__?: any;
+} & { [key: string]: unknown };
+
+export default function _get<T extends Object, P extends string | symbol>(
+  this: unknown,
+  ...args: [target: T, property: P, receiver?: unknown]
+): P extends keyof T ? T[P] : any {
+  let _getImpl = _get;
```

Comment:
Let's keep the original implementation where we reassign `_get`, and add a `@ts-expect-error` to it.

This is different from the original one because the feature detection would run every single time `_get` is called, rather than just the fist one.

Also for the other helpers.